test/docfx.Test/lib/HrefUtilityTest.cs

```diff
@@ -10,13 +10,13 @@ public static class HrefUtilityTest
         [Theory]
         [InlineData("", "", "", "")]
         [InlineData("a", "a", "", "")]
-        [InlineData("a#b", "a", "", "#b")]
-        [InlineData("a#b<", "a", "", "#b<")]
-        [InlineData("a?c", "a", "?c", "")]
-        [InlineData("a?b#c", "a", "?b", "#c")]
-        [InlineData("a#b?c=d", "a", "", "#b?c=d")]
-        [InlineData("a?b#c?d=e", "a", "?b", "#c?d=e")]
-        [InlineData("a?b#c#d", "a", "?b", "#c#d")]
+        [InlineData("a#b", "a", "", "b")]
+        [InlineData("a#b<", "a", "", "b<")]
+        [InlineData("a?c", "a", "c", "")]
+        [InlineData("a?b#c", "a", "b", "c")]
+        [InlineData("a#b?c=d", "a", "", "b?c=d")]
+        [InlineData("a?b#c?d=e", "a", "b", "c?d=e")]
+        [InlineData("a?b#c#d", "a", "b", "c#d")]
         public static void SplitHref(string href, string path, string query, string fragment)
```

Comment:
A fragment is an internal page reference, sometimes called a named anchor. It usually appears at the end of a URL and **begins with a hash (#) character followed by an identifier**. It refers to a section within a web page